js/blocks/controls/repeater.js

```diff
@@ -10,14 +10,28 @@ const { BaseControl, IconButton } = wp.components;
 import { RepeaterRows } from '../components';
 
 const BlockLabRepeaterControl = ( props ) => {
-	const { field, getValue, onChange, parentBlock, parentBlockProps } = props;
+	const { field, onChange, parentBlock, parentBlockProps } = props;
 	const { attributes, setAttributes } = parentBlockProps;
 	const attr = { ...attributes };
 	const value = attr[ field.name ];
 	const defaultRows = [ {} ];
 	const hasRows = value && value.hasOwnProperty( 'rows' );
 	const rows = hasRows ? value.rows : defaultRows;
 
+	/**
+	 * Adds a new empty row, using { '': '' }.
+	 *
+	 * Simply using {} results in <ServerSideRender> not sending an empty row,
+	 * and the empty row isn't rendered in the editor.
+	 *
+	 * @see https://github.com/getblocklab/block-lab/issues/393
+	 */
+	const addEmptyRow = () => {
+		const withAddedRow = rows.concat( { '': '' } );
```

Comment:
This isn't ideal, but it's worked so far in brief testing. I'll keep testing this.